iothub_client/samples/pnp/pnp_simple_thermostat/pnp_simple_thermostat.c

```diff
@@ -0,0 +1,484 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// This sample implements a PnP based thermostat.  This demonstrates a *relatively* simple PnP device
+// that does only acts as a thermostat and does not have additional components.  
+
+// The DigitalTwin Definition Language document describing the component implemented in this sample
+// is available at https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json.
+
+// Standard C header files
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+// IoTHub Device Client and IoT core utility related header files
+#include "iothub.h"
+#include "iothub_device_client.h"
+#include "iothub_message.h"
+#include "azure_c_shared_utility/threadapi.h"
+#include "azure_c_shared_utility/xlogging.h"
+
+// JSON parser library
+#include "parson.h"
+
+// pnp_device_client_helpers.h is part of the PnP sample code and helps in 
+// creation of the IOTHUB_DEVICE_CLIENT_HANDLE
+#include "pnp_device_client_helpers.h"
```

Comment:
to avoid any potential confusion, place this function back in the main file.

Reply:
fixed.